test/mailers/previews/last_synced_mailer_preview.rb

```diff
@@ -1,4 +1,17 @@
 class LastSyncedMailerPreview  < ActionMailer::Preview
-    def last_synced_more_than_3_days
+    require 'mailer_service/last_seen_last_sync_service'
+    def summary_of_last_synced
+        result = LastSeenLastSyncService.processed_data
+
+        if result[:last_activity].present? && result[:last_activity][:districts].any?
+            LastSyncedMailer.summary_of_last_synced(result[:last_activity]).deliver_now
```

Comment:
Remove the .deliver_now for it so that an email and valid smtp details are not required with doing the preview.
```suggestion
            LastSyncedMailer.summary_of_last_synced(result[:last_activity])
```